src/machines/movie.ts

```diff
@@ -0,0 +1,127 @@
+import { assign, createMachine } from 'xstate';
+
+import { OfferType } from '@/constants/OfferType';
+import type { Values } from '@/types/Values';
+
+const Themes = {
+  'Actie- en avonturenfilm': '1.7.2.0.0',
+  'Animatie en kinderfilms': '1.7.12.0.0',
+  'Documentaires en reportages': '1.7.1.0.0',
+  'Griezelfilm of horror': '1.7.6.0.0',
+  'Historische film': '1.7.8.0.0',
+  Komedie: '1.7.3.0.0',
+  Kortfilm: '1.7.13.0.0',
+  Filmmusical: '1.7.10.0.0',
+  Drama: '1.7.4.0.0',
+  'Science fiction': '1.7.7.0.0',
+  Cinefiel: '1.7.11.0.0',
+  'Meerdere filmgenres': '1.7.14.0.0',
+  Thriller: '1.7.15.0.0',
+} as const;
+
+const Types = {
```

Comment:
Shouldn't this be in a more general file? Since movie will probably always be of type  `Film` ?